WISE-Community/WISE-Client
# refactor(UtilService): Move is[Image/Video/Audio]()

## Changes
- Move UtilService.is\[Image/Video/Audio\]() functions to own file
- Add RegEx to ensure extension matches are at end of filename string
- Add tests

## Test
- Make sure these tasks work as before:
  - Upload files (images) to student assets in a run (via Discussion) and use in discussion post
  - Upload and preview files in Authoring Tool File Manager
  - Inset files to HTML component using TinyMCE and File Manager

Closes #1000
